src/shims/ApiCompatBaseline.netcoreapp.netstandard.txt

```diff
@@ -67,4 +67,14 @@ MembersMustExist : Member 'System.Linq.EnumerableQuery..ctor()' does not exist i
 MembersMustExist : Member 'System.Threading.Tasks.TaskExtensions.ConfigureAwait(System.IAsyncDisposable, System.Boolean)' does not exist in the implementation but it does exist in the contract.
 MembersMustExist : Member 'System.Threading.Tasks.TaskExtensions.ConfigureAwait<T>(System.Collections.Generic.IAsyncEnumerable<T>, System.Boolean)' does not exist in the implementation but it does exist in the contract.
 MembersMustExist : Member 'System.Threading.Tasks.TaskExtensions.WithCancellation<T>(System.Collections.Generic.IAsyncEnumerable<T>, System.Threading.CancellationToken)' does not exist in the implementation but it does exist in the contract.
-Total Issues: 59
+TypesMustExist : Type 'System.IAsyncDisposable' does not exist in the implementation but it does exist in the contract.
```

Comment:
These are here because we aren't binplacing into `netcoreapp` (versionless).  We could do that too.